synapse/handlers/profile.py

```diff
@@ -286,13 +286,19 @@ async def set_avatar_url(
                 400, "Avatar URL is too long (max %i)" % (MAX_AVATAR_URL_LEN,)
             )
 
+        avatar_url_to_set = new_avatar_url  # type: Optional[str]
```

Comment:
Do we need similar changes to the display name?

Reply:
No, displayname has this lines already.

https://github.com/matrix-org/synapse/blob/4218473f9ea6a2680c21e96368dfe9c06271c8a4/synapse/handlers/profile.py#L201-L203

Reply:
Ah, OK that makes sense. :)